tests/integration/standard/test_query.py

```diff
@@ -46,7 +47,7 @@ def setup_module():
     if not USE_CASS_EXTERNAL:
         use_singledc(start=False)
         ccm_cluster = get_cluster()
-        ccm_cluster.clear()
+        ccm_cluster.stop()
```

Comment:
Other tests use `.stop()` and I see no reason why we need to clear data here. It was preventing me from the running this module in isolation, because if you call `.clear()` with no pre-existing cluster data it'll throw an exception.